CHANGELOG.next.asciidoc

```diff
@@ -354,6 +352,8 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Add ExpandEventListFromField config option in the kafka input. {pull}13965[13965]
 - Add ELB fileset to AWS module. {pull}14020[14020]
 - Add module for MISP (Malware Information Sharing Platform). {pull}13805[13805]
+- Add support for gzipped files in S3 input {pull}13980[13980]
```

Comment:
These two changelog entries were at the wrong session, moving to the correct session in this PR.